testbed/testbed/scripts/pdf_composition.py

```diff
@@ -0,0 +1,24 @@
+import process
+import os
+import sys
+import utils
+from matplotlib.backends.backend_pdf import PdfPages
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+print(dir_path)
+name = sys.argv[1]
+filename = "/" + name + ".pdf"
+
+print(filename)
+
+with PdfPages(dir_path + "/../../compositions/results/"+filename) as export_pdf:
```

Comment:
Why not adding this to `pdf.py` instead of having two different scripts?

Reply:
Do you mean refactor the pdf.py file to take the paths as args ?

Reply:
Yep! Instead of receiving just the filename, add the absolute path. That way we don't have different scripts with a lot of redundant code.